### PR TITLE
Fix for Google Analytics script to be executed properly

### PR DESCRIPTION
**Pull request by asavin**
### Motivation

- #359 

GA tracking is currently broken on master. This PR fixes that.

### Test plan

- Open site. Open Chrome devtools. Make sure that `ga.js` is properly loaded among the rest of JS files.

NOTE that analytics is disabled on staging and preview app, so you can only test this on live env (or local dev env where you specifically enable analytics).

### Pre-merge checklist

- [ ] Documentation
- [ ] Unit tests
- [ ] Reviews
  - [ ] Code review
  - [ ] Design review
- [ ] Manual testing
  - [ ] Windows 7 IE 11
  - [ ] Windows 10 Edge
  - [ ] Mac OS El Capitan Safari
  - [ ] Mac OS El Capitan Chrome
  - [ ] Mac OS El Capitan Firefox
  - [ ] iOS 9 Safari
  - [ ] Android 6 Chrome
  - [ ] Listen to the site on a screen reader
  - [ ] Navigate the site using the keyboard only
- [ ] Tester approved

### CODE REVIEW

File: site/index.ejs [hunk 1]
```diff
@@ -32,15 +32,15 @@
     <title><% if (typeof title !== 'undefined') {%><%= title %><% } %></title>
     <% if (typeof cssPath !== 'undefined') { %><link rel="stylesheet" href="<%= cssPath %>" type="text/css" charset="utf-8"><% } %>
     <% if (typeof tracking !== 'undefined' && tracking) { %>
-    <script type="text/javascript" src="https://secure.leadforensics.com/js/77932.js" /><noscript><img src="https://secure.leadforensics.com/77932.png" style="display: none;" /></noscript>
+    <script type="text/javascript" src="https://secure.leadforensics.com/js/77932.js"></script><noscript><img src="https://secure.leadforensics.com/77932.png" style="display: none;" /></noscript>
```
Comment (asavin) [inline site/index.ejs]: Each `<script>` tag must have a separate `</script>` closing tag - they are an exception and cannot self close.

**Comment by dstodolny** [comment]
However I'm not completely sure about your solution as there is no gif in the PR

**Comment by asavin** [comment]
giphy 3 2

**Comment by asavin** [comment]
Gif added

**Comment by dstodolny** [comment]
Accept

**Pull request merged**